build_system/src/main.rs

```diff
@@ -29,6 +30,7 @@ Available commands for build_system:
     prepare  : Run prepare command
     build    : Run build command
     test     : Run test command
+    info:    : Run info command
```

Comment:
At this point, I'm starting to believe we should give better explanation of what each command does here.
But this can be done in a later PR.